.github/workflows/build-macos.yaml

```diff
@@ -11,13 +11,13 @@ jobs:
     strategy:
       matrix:
         go-version: [1.19.x]
-        platform: [macos-latest]
+        platform: [macOS-12]
```

Comment:
12 is the latest they support, `latest` currently points to 11, but they plan on updating it to 12 and are giving a warning to discourage users from using the `latest` tag.